backend/editor/entries.py

```diff
@@ -120,10 +120,12 @@ async def import_from_github(self, description):
                 status = await self.parse_taxonomy(filepath)  # Parse the taxonomy
 
             async with TransactionCtx():
-                await self.create_project(description)  # Creates a "project node" in neo4j
+                await self.set_project_status(status="OPEN")
 
             return status
         except Exception as e:
+            async with TransactionCtx():
+                await self.set_project_status(status="FAILED")
```

Comment:
Maybe you could set project status FAILED only if create_project did not fail ? (create project could return created project and you could test it here, remember to create the variable with `None` before the `try` though to be sure it exists.